pkg/rules/rules.go

```diff
@@ -16,12 +16,13 @@ type Urlregexp struct {
 
 // Rules struct fail2ban config.
 type Rules struct {
-	Bantime    string      `yaml:"bantime"`  // exprimate in a smart way: 3m
-	Enabled    bool        `yaml:"enabled"`  // enable or disable the jail
-	Findtime   string      `yaml:"findtime"` // exprimate in a smart way: 3m
-	Maxretry   int         `yaml:"maxretry"`
-	Urlregexps []Urlregexp `yaml:"urlregexps"`
-	StatusCode string      `yaml:"statuscode"`
+	Bantime        string      `yaml:"bantime"`  // exprimate in a smart way: 3m
+	Enabled        bool        `yaml:"enabled"`  // enable or disable the jail
+	Findtime       string      `yaml:"findtime"` // exprimate in a smart way: 3m
+	Maxretry       int         `yaml:"maxretry"`
+	Urlregexps     []Urlregexp `yaml:"urlregexps"`
+	StatusCode     string      `yaml:"statuscode"`
+	SourceIPHeader string      `json:"sourceIpHeader,omitempty" toml:"sourceIpHeader,omitempty,omitempty" yaml:"sourceIpHeader"`
```

Comment:
```suggestion
	SourceIPHeader string      `yaml:"sourceIpHeader"`
```

Only yaml is usefull